checker/src/main/java/org/checkerframework/checker/initialization/InitializationAnnotatedTypeFactory.java

```diff
@@ -140,19 +146,21 @@ protected InitializationAnnotatedTypeFactory(BaseTypeChecker checker) {
         unknownInitializationValueElement =
                 TreeUtils.getMethod(UnknownInitialization.class, "value", 0, processingEnv);
 
-        Set<Class<? extends Annotation>> tempInitAnnos = new LinkedHashSet<>(4);
+        Set<Class<? extends Annotation>> tempInitAnnos = new LinkedHashSet<>(5);
         tempInitAnnos.add(UnderInitialization.class);
         tempInitAnnos.add(Initialized.class);
         tempInitAnnos.add(UnknownInitialization.class);
         tempInitAnnos.add(FBCBottom.class);
+        tempInitAnnos.add(PolyInitialized.class);
 
         initAnnos = Collections.unmodifiableSet(tempInitAnnos);
 
-        Set<String> tempInitAnnoNames = new HashSet<>(4);
+        Set<@CanonicalName String> tempInitAnnoNames = new HashSet<>(5);
```

Comment:
Did you get an error without this annotation? If not, we should make sure the `Signature Checker` runs on this code.

Reply:
No, there is no error. However, when I run use Signature Checker, most symbols could not be found and I got 197 errors right now. I will try to figure out what is going on.

Reply:
Let's fix that as a separate issue - please open one.

Reply:
Verified that the signature checker run correctly.